<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BestoPOSoClass</class>
 <widget class="QWidget" name="BestoPOSoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BestoPOSo</string>
  </property>
  <widget class="QPushButton" name="createItemButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Item Creator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="POSModeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Open POS Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endOfDayButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>End Of Day</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BestoPOSo.qrc"/>
 </resources>
 <connections/>
</ui>
